examples/pub-sub-rewind/javascript/src/script.ts

```
@@ -0,0 +1,195 @@
+    });
+
+    const markets = ['homeWin', 'draw', 'awayWin', 'nextGoal'];
+    const numMarketsToUpdate = Math.floor(Math.random() * 3) + 1;
+    const marketsToUpdate = markets.sort(() => 0.5 - Math.random()).slice(0, numMarketsToUpdate);
+
+    const newOdds = { ...matchData };
+
+    marketsToUpdate.forEach((market) => {
+      if (market === 'nextGoal') {
+        const team = Object.keys(newOdds.match.nextGoal)[Math.floor(Math.random() * 3)];
+        newOdds.match.nextGoal[team] = (parseFloat(newOdds.match.nextGoal[team]) + (Math.random() * 0.2 - 0.1)).toFixed(2);
```

ESLint

[error] Replace `2` with `⏎··········2,⏎········`

(prettier/prettier)